src/router/index.js

```diff
@@ -105,11 +127,21 @@ const router = new Router({
 			props: route => ({
 				albumName: route.params.albumName,
 			}),
+			meta: {
+				rootTitle: (to) => {
+					return `${t('photos', 'Album')} ${to.params.albumName}`
```

Comment:
You are concatenating a translated string. Always use placeholders inside the translation string instead.

```suggestion
					return t('photos', 'Album {title}', { title: to.params.albumName })
```